cyto_dl/train.py

```diff
@@ -97,9 +97,11 @@ def train(cfg: DictConfig) -> Tuple[dict, dict]:
         log.info("Starting training!")
 
         if cfg.get("weights_only"):
+            assert cfg.get(
+                "ckpt_path"
```

Comment:
Nice!

Any unit tests that exercise this?

Reply:
manual testing only at the moment... at the moment I'm reluctant to add more unit tests when we're already timing out on windows.